documentation.md

```diff
@@ -205,3 +205,9 @@ following = user.following() #user var taken from above
 print(following)
 ```
 ---
+
+## Gitlab
+
+```python
+from scrape_up import gitlab
```

Comment:
Do not just mention the class but also define all the methods used as shown in the example below
![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/33c94500-d0a9-4224-9209-d87191ed1acc)